app/views/federations/show.json.jbuilder

```diff
@@ -0,0 +1,12 @@
+
+@federated_results[:more_results_label] = t('global.actions.view-more-at') + @foederati_provider.display_name
```

Comment:
This doesn't seem quite the right approach to a jbuilder view. I'd expect it not to modify `@federated_results` and to make fuller use of the jbuilder DSL, like:
```ruby
json.total @federated_results[:total]
json.more_results_label t('global.actions.view-more-at') + @foederati_provider.display_name
json.more_results_url format(@foederati_provider.urls.site, query: @query)
json.tab_subtitle [number_with_delimiter(@federated_results[:total]), t('site.results.results')].join(' ')
json.search_results @federated_results[:results] do |result|
  json.title result[:title]
  json.img { src: result[:thumbnail] } if result[:thumbnail]
  json.object_url result[:url]
end
```

Also, the present implementation leaves blank `:thumbnail` entries on `:search_results`, which the above would not.